index.html

```diff
@@ -0,0 +1,110 @@
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="index.html">Feed</a>
+                        </li>
+                    </ul>
+                    <div class="container-fluid d-flex justify-content-lg-end">
+                        <form class="d-flex" role="search">
+                            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                            <button class="btn btn-outline-success" type="submit">Search</button>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <div class="container d-flex flex-column align-items-center text-center">
+            <form action="/profile.html" class="mt-5">
+                <h2 class="h5">Share something:</h2>
+                <div class="mb-3">
+                    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+                </div>
+                <button class="btn btn-primary btn-lg">Post</button>
+            </form>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
+                <div class="card-body">
+                    <p class="card-text">Happy birthday, you son of a gun!</p>
+                    <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend5.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
+                <div class="card-body">
+                    <p class="card-text">Congratulations!</p>
+                    <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend3.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
+                <div class="card-body">
+                    <p class="card-text">Getting old! Although the alternative is worse.. HBD :D</p>
+                    <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
```

Comment:
same thing here and in other places

Reply:
fixed

Reply:
Sorry,  can't see any change. I suggested you add a valid value to the alt attribute. Like
```
<img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="image of friend4">
```

As you can see above, I set the value of the alt attribute to `image of friend4`.